src/react/deckgl.js

```diff
@@ -21,10 +21,10 @@ import React, {PropTypes} from 'react';
 import autobind from 'autobind-decorator';
 import WebGLRenderer from './webgl-renderer';
 import {LayerManager} from '../lib';
-import {EffectManager} from '../lib';
+import {EffectManager} from '../experimental/lib';
 import {GL, addEvents} from 'luma.gl';
 
-import {ReflectionEffect} from '../effects';
+import {ReflectionEffect} from '../experimental/effects';
```

Comment:
I still think you should add a new prop instead, and let the application or example specify the effects.

In this file:
L35: ```effects: PropTypes.array,```
L47:```effects: [],```